File: aspnet-core/src/ERP.Application.Shared/SupplyChain/Inventory/ICLOT/Dtos/ICLOTDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.SupplyChain.Inventory.ICLOT.Dtos
{
    public class ICLOTDto : EntityDto
    {
        public int? TenantID { get; set; }

        public string LotNo { get; set; }

        public DateTime? ManfDate { get; set; }

        public DateTime? ExpiryDate { get; set; }

        public bool Active { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

    }
}

File: angular/src/app/main/supplyChain/inventory/shared/dto/icLot-dto.ts
export class ICLOTDto  {
    id:number|undefined;
    tenantID?: number|undefined;
    lotNo: string;
    manfDate?: string;
    expiryDate?: string;
    active: boolean;
  
}
export class GeticLotForViewDto {
    icLotDto: ICLOTDto
}
export class GeticLotEditOutput {
    iclot: ICLOTDto
}

export class PagedResultDtoiclot {
    totalCount: number;
    items: GeticLotForViewDto[]
}
export class CreateOrEditLotNoDto {
    
    id:number|undefined;
    tenantID?: number|undefined;
    lotNo: string;
    manfDate?: string|Date|null;
    expiryDate?: string|Date|null;
    active: boolean;
    

}